tests.js

```diff
@@ -11,8 +11,56 @@ chai.use(chaiChange)
 /*1: Guess The Number
 Write a program that will generate a number between 1 and 100. Your program will ask the user to guess the generated number.When the user types a guess, the program should tell them if they won (the guess matched the computer's number), if their guess was higher than the number the computer generated, OR if the guess was lower than the number the computer generated. Terminate the program if the user types "exit". */
 
+1.
+describe('compuGuess()', () => {
+  it('should not generate a number above 100', () => {
+    console.log('Guess the generated number');
+    expect()).to.equal('1-100')
+
+  it('should tell them if they won')
+  })
+})
+it("should tell them if ")
+
+describe('compuGuess()', () => {
+  it('should not add two numbers', () => {
```

Comment:
This test isn't a reasonable test case. The `getComputerGuess()` function doesn't do a lot of things (adding, subtracting, etc.), and it'd be asking too much to test all these cases. Stick to testing the requirements of the spec first, and then consider cases of invalid input.